public/scripts/ssooverride.js

```diff
@@ -7,6 +7,21 @@
   var hostname = document.getElementById("ssooverride").getAttribute("data-login-hostname");
   var loginToSave = document.getElementById("ssooverride").getAttribute("data-login-to-save");
 
+  // XXXNew Thimble
+  // As soon as the page has loaded it, we attach listeners to hide the
+  // "new thimble" bar:
+  var closeBannerButton = document.getElementById("new-thimble-banner");
+  function hideBanner(e) {
+    e.preventDefault();
+    e.stopPropagation();
```

Comment:
that might be cause using `onclick` is an ancient instant-event-handling, with different rules from actual event listeners. If you want `stopPropagation` to work, use `addEventListener` and `removeEventListener`, not `onclick`